app/_src/policies/meshtrafficpermission.md

```diff
@@ -7,6 +7,13 @@ This policy uses new policy matching algorithm.
 Do **not** combine with [TrafficPermission](/docs/{{ page.release }}/policies/traffic-permissions).
 {% endwarning %}
 
+{% tip %}
+Remember [Mutual TLS](/docs/{{ page.release }}/policies/mutual-tls) has to be enabled to make MeshTrafficPermission works.
```

Comment:
"Remember Mutual TLS has to be enabled to make MeshTrafficPermission works" sounds kind of weird. Sounds like it's missing comma or "remember" is redundant. How about 
```
mTLS has to be enabled to make MeshTrafficPermission work.
```
or
```
MeshTrafficPermission requires enabled mTLS in the Mesh.
```